censusdis/symbolic.py

```diff
@@ -0,0 +1,95 @@
+# Copyright (c) 2022 Darren Erik Vengroff
+
+"""
+Utilities for creating symbolic names.
+
+This module processes data sets from the US Census
+and their respective symbolic names for
+documentation purposes.
+"""
+
+import os
+import censusdis.data as ced
+import censusdis.datasets
+
+class symbolic:
+    def __init__(self):
+        self.dictionary = {}
+        self.module_message = [
+            "# Copyright (c) 2022 Darren Erik Vengroff\n",'"""\n'
+            "This module contains abbreviated names for commonly used data sets.\n",
+            "These are typically used as the first argument to :py:func:`censudis.data.download`.\n",
+            "There are a lot more data sets available than there are symbolic names here.\n",
+            "But you can always use raw strings. For example, even for `ACS5` you can use\n",
+            "`acs/acs5` instead.\n", '"""\n' 
+        ]
+
+    def store_dataset(self, dataset_list: list, url_list: list):
```

Comment:
I'd like to see a docstring here to explain this method. When I first saw the name I thought this was storing to the output file, but I see that it's actually kind of merging two lists together.